disableDetective.py

```diff
@@ -277,15 +291,16 @@ def delete_members(d_client: botocore.client.BaseClient, graph_arn: str,
             d_client = master_session.client('detective', region_name=region)
             graphs = get_graphs(d_client)
             if not graphs:
-                logging.info(f'Amazon Detective is NOT disabled in {region}')
-                continue
-            logging.info(f'Amazon Detective is disabled in region {region}')
+                logging.info(f'Amazon Detective has already been disabled in {region}')
+            else:
+                logging.info(f'Disabling Amazon Detective in region {region}')
 
             try:
-
                 for graph in graphs:
-                    delete_members(
-                        d_client, graph, aws_account_dict)
+                    if not args.delete_graph:
+                        delete_members(d_client, graph, aws_account_dict)
+                    else:
+                        d_client.delete_graph(graph)
```

Comment:
I'm not super familiar with the different ways that the Python SDK can be invoked, but do we need to use `delete_graph(GraphArn=graph)` here as we're doing with other API calls? Or is the graph ARN the only parameter to this method?

Reply:
It is the only parameter. You can see the documentation here: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/detective.html#Detective.Client.delete_graph